### PR TITLE
Issues 87 and 72

### PR DESCRIPTION
**Pull request by Burtchen**
This should address two issues:

- issue #87 : make the initial message on iOS less confusing by going for our core "Share why YOUR vote matters" rather than "oops" right from the beginning - after all, the next thing the user sees is an option to upload/take a picture as provided by iOS
- issue #72 when checking for canvas resizing, also check whether controls (buttons, inputs...) are visible - if not, which should be edge cases, put them in the top of the canvas area

Things to consider:
- Some real-life iOS testing would be great
- I don't think it makes too much sense now, but the controls re-positioning function could also have an option to reset items to their original spot if the user has enlarged the window

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -262,10 +279,14 @@
     ui.hide(document.querySelector("#canvas"));
     wrapper.classList.add('camera-failure');
     wrapper.classList.add('fgwhite');
-    var textNode = document.createTextNode('Oops! It looks like your camera won\'t work here, but you can upload a photo instead by pressing the button below.');
-    wrapper.appendChild(textNode);
-
-    document.querySelector('#streaming .text').textContent = 'Upload photo';
+    var noCameraText = iOS ? 'Share why YOUR vote matters!' : 'Oops! It looks like your camera won\'t work here, but you can upload a photo instead by pressing the button below.';
```
Comment (dglazkov) [inline js/main.js:290]: For camera-less computers, the message is flowing underneath the button here.

File: js/main.js [hunk 2]
```diff
@@ -296,6 +317,11 @@
     savebutton.value = 'Save';
     ui.show(document.querySelector('#savebutton'));
     shareTarget.captureImage();
+    video.pause();
+    if (localstream && typeof localstream.getTracks === "function") {
```
Comment (dglazkov) [inline js/main.js:330]: Where does `localstream` come from?

**Review by dglazkov — CHANGES_REQUESTED** [review]
I took a quick look, a couple of comments.

File: js/main.js (hunk 2 above)
Comment (Burtchen) [reply js/main.js:330]: Localstream is assigned from `stream`  in the `getUserMedia` callback and initially defined as `null`. This is how we actually stop the camera itself.

**Comment by Burtchen** [comment]
Issues should be fixed now.

**Pull request merged**